packages/at_client/lib/src/transformer/request_transformer/notify_request_transformer.dart

```diff
@@ -50,7 +50,8 @@ class NotificationRequestTransformer
       ..priority = notificationParams.priority
       ..strategy = notificationParams.strategy
       ..latestN = notificationParams.latestN
-      ..notifier = notificationParams.notifier;
+      ..notifier = notificationParams.notifier
+      ..ttln = notificationParams.notificationExpiry.inMilliseconds;
```

Comment:
Do we have tests which exercise the ttln functionality, and can we perhaps write a functional test to verify this (since we've just added it to the transformer?)

Reply:
@gkc: Updated a functional test to verify ttln

Reply:
Thank you @sitaram-kalluri